### PR TITLE
CB-20135:Turn off KMS validations / allows keys in remote AWS account

### PR DESCRIPTION
**Pull request by satyabollineni**
PR created for master : https://github.com/hortonworks/cloudbreak/pull/14182/files

### CODE REVIEW

File: environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EncryptionKeyArnValidator.java [hunk 1]
```diff
@@ -104,9 +104,12 @@ public ValidationResult validate(EnvironmentValidationDto environmentValidationD
...
             List<String> keyArns = encryptionKeys.getCloudEncryptionKeys().stream().map(CloudEncryptionKey::getName).collect(Collectors.toList());
             if (keyArns.stream().noneMatch(s -> s.equals(encryptionKeyArn))) {
-                validationResultBuilder.error("Following encryption keys are retrieved from the cloud " + keyArns +
-                                " . The provided encryption key " + encryptionKeyArn +
-                        " does not exist in the given region's encryption key list for this credential.");
+                validationResultBuilder.warning("Following encryption keys are retrieved from the cloud " + keyArns +
+                        " . The provided encryption key " + encryptionKeyArn +
+                        " does not exist in the given region's encryption key list for this credential." +
+                        " This is possible if the key is present in a different AWS Account." +
+                        " Please ensure that the Key is present and have valid permissions otherwise would result to failures in EBS volume creation");
```
Comment (lajosrodek) [inline environment/src/main/java/com/sequenceiq/environment/environment/validation/validators/EncryptionKeyArnValidator.java:111]: Nit:
* Insert `it` before `would`.
* Replace `to failures in` with `in failures with`.
* Add `.` at the end of the last sentence (after `creation`).

**Review by lajosrodek — CHANGES_REQUESTED** [review]
Tiny asks.

**Review by lajosrodek — APPROVED** [review]
Thanks, LGTM.

**Comment by satyabollineni** [comment]
The swagger compatibility failures are expected as per this thread : https://cloudera.slack.com/archives/CF66M7WP6/p1675094183296449

**Pull request merged**